### ISSUE TITLE
The official docker "latest" image is still running 3.0.3

### ISSUE THREAD

**Issue by yourpropertyexpert** [Codeception/Codeception#5898]
#### What are you trying to achieve?

I'm trying to run codeception in docker

#### What do you get instead?

> Provide console output if related. Use `-vvv` mode for more details.

```bash

latest: Pulling from codeception/codeception
Digest: sha256:3662f9f9d2f6ab743c81fc9e3e9725f1a81f1403a0186548002fb16f1e11a107
Status: Image is up to date for codeception/codeception:latest
[...]
Codeception PHP Testing Framework v3.0.3


```
> Provide test source code if related

```php
// paste test
```
### Details

* Codeception version: [That's the issue]
* PHP Version: [N/A]
* Operating System: [N/A]
* Installation type: Docker
* List of installed packages: N/A
* Suite configuration:

```yml
# paste suite config here
```

**Comment by yourpropertyexpert** [comment]
I've added a docker image to my repos at:
[]((https://hub.docker.com/repository/docker/yourpropertyexpert/codeception4))

The DockerFile is:


> FROM php:7.3-apache
> 
> ADD https://raw.githubusercontent.com/mlocati/docker-php-extension-installer/master/install-php-extensions /usr/local/bin/
> 
> RUN chmod uga+x /usr/local/bin/install-php-extensions && sync
> 
> RUN apt-get update
> RUN apt-get install -y git
> RUN apt install zip unzip
> 
> RUN install-php-extensions zip
> 
> COPY composer.json /var/www
> 
> RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
> RUN cd /var/www && composer update
> 


The composer.json is:

> {
>   "require": {
>     	"codeception/codeception": "^4.0",
>         "codeception/module-phpbrowser": "^1.0.0",
>         "codeception/module-asserts": "^1.0.0",
>         "codeception/module-webdriver": "^1.0.0"
>   }
> }
>

**Comment by Naktibalda** [comment]
Fixed by #5917
Please test codeception/codecetion:4.1.5
